src/openforms/emails/templates/emails/templatetags/product_information.html

```diff
@@ -0,0 +1 @@
+{% if product and product.information %}{{ product.information|safe }}{% endif %}
```

Comment:
This is an XSS attack vector.

Content editors writing down the production information can effectively include XSS snippets, which will run in the context of the end-user's browser.

Please use https://pypi.org/project/django-bleach/ and sanitize the content of this field - using the tmeplate filter should be sufficient.

Reply:
picking this up as part of https://github.com/open-formulieren/security-issues/issues/4

Reply:
We moved this to a new task